simtools/array_simulator.py

```diff
@@ -0,0 +1,320 @@
+
+import logging
+import os
+from pathlib import Path
+from copy import copy
+
+import astropy.units as u
+
+import simtools.config as cfg
+import simtools.io_handler as io
+import simtools.util.general as gen
+from simtools.simtel.simtel_runner_array import SimtelRunnerArray
+from simtools.model.array_model import ArrayModel
+
+__all__ = ['ArraySimulator']
+
+
+class MissingRequiredEntryInArrayConfig(Exception):
+    pass
+
+
+class ArraySimulator:
+    '''
+    ArraySimulator is responsible for managing simulation of array of telescopes. \
+    It interfaces with simulation software-specific packages, like sim_telarray.
+
+    The configuration is set as a dict configData or a yaml \
+    file configFile. An example of configData can be found \
+    below.
+
+    .. code-block:: python
+
+    configData = {
+        'simtelDataDirectory': '.',
```

Comment:
Is the data directory really the local one? Maybe it's better to give in the example something more realistic like `data/ `

Reply:
Right. I will change it. Anyway, it's just an example.

Reply:
OK!